azureLogIngestion/adapter/opentelemetry/index.ts

```diff
@@ -1,6 +1,18 @@
 import { Context } from "@azure/functions"
+import opentelemetry, { SpanStatusCode, SpanKind, TraceFlags } from "@opentelemetry/api"
 import { BasicTracerProvider, BatchSpanProcessor } from "@opentelemetry/tracing"
 import { CollectorTraceExporter } from "@opentelemetry/exporter-collector"
+import { timeStampToHr, endTimeHrFromDuration, convertToMs } from "../utils/time"
+
+interface Records {
```

Comment:
We have code like this in a bunch of places. We should probably centralize it somewhere.

Reply:
In a branch I'd archived, I moved the utils up to be easily shared by both processors; I can restore that.